odoo/cli/command.py

```diff
@@ -23,19 +26,61 @@ def __init_subclass__(cls):
 
 Use '{odoo_bin} <command> --help' for individual command help."""
 
+
 class Help(Command):
     """ Display the list of available commands """
     def run(self, args):
-        padding = max([len(cmd) for cmd in commands]) + 2
+        commands = list_commands()
+        padding = max(len(cmd) for cmd in commands) + 2
         command_list = "\n    ".join([
             "    {}{}".format(name.ljust(padding), (command.__doc__ or "").strip())
-            for name, command in sorted(commands.items())
+            for name in sorted(commands)
+            if (command := find_command(name))
         ])
-        print(ODOO_HELP.format(  # pylint: disable=bad-builtin
+        print(ODOO_HELP.format(  # pylint: disable=bad-builtin  # noqa: T201
             odoo_bin=Path(sys.argv[0]).name,
             command_list=command_list
         ))
 
+
+def list_commands(from_internal=True, from_addons=True) -> list[str]:
```

Comment:
I think that if you can avoid hardcoding do so. Having a list means that you need to maintain it and are coupling changes.
I have split that function into 2 to avoid boolean flags. So that change would just replace a 5 line function called once when listing commands in help by a hardcoded list  outside of that file.